src/seedu/addressbook/data/person/Name.java

```diff
@@ -43,6 +43,16 @@ public static boolean isValidName(String test) {
         return Arrays.asList(fullName.split("\\s+"));
     }
 
+    /**
+     * Compares two strings and determines which is lexographically larger.
+     *
+     * @return 1, 0 or -1 if this fullName is larger, equal, or smaller than other fullName, respectively.
+     */
+    public int compareTo(Name other) {
+        return this.fullName.compareTo(other.fullName);
+    }
+
```

Comment:
Stray line break? It is a good practice to check the diff of the PR and fix any problems before submitting it.